### DISCUSSION TITLE
高亮绘制失效

### DISCUSSION THREAD

**Discussion by BraveGlide** [tyron12233/CodeAssist#450] [Q&A]
问题：语法高亮部分界面绘制失效，高亮失效界面无法编辑文本

![Screenshot_20220619_112830_com.tyron.code.jpg](https://user-images.githubusercontent.com/73690771/174464603-bef5bd0f-92b0-4b70-bd4f-41281e7ddcce.jpg)

**Comment by Pigletzzz** [comment]
I have the same problem with keyboard input on my tablet.
